### PR TITLE
Commenting soccer player article extraction script 

### PR DESCRIPTION
**Pull request by abhishekikc**
@marfox I have commented the code. I have few doubts:
a) SPARQL query output contains two columns-URL and id. While the load_wiki_ids just consider it to contain ids.
b)Copied file to output directory also seems to give the whole file rather then just the extracted portion of soccer_articles. 
Should I proceed to make the above changes.

### CODE REVIEW

File: get_soccer_players_articles.py [hunk 1]
```diff
@@ -9,23 +9,27 @@
 
 DEBUG = True
 
-
+#Returns a list of Wiki IDs read from input file path "filein"
```
Comment (marfox) [inline get_soccer_players_articles.py]: Please use *docstrings* to document functions

**Comment by marfox** [comment]
On 3/13/15 7:08 AM, Abhishek Kumar Tiwari wrote:
> @marfox <https://github.com/marfox> I have commented the code. I have
> few doubts:
> a) SPARQL query output contains two columns-URL and id. While the
> load_wiki_ids just consider it to contain ids.
Yes, I think I grepped the IDs only from the SPARQL output, that's why.
> b)Copied file to output directory also seems to give the whole file
> rather then just the extracted portion of soccer_articles.
What do you mean? It should copy the whole article.

**Comment by abhishekikc** [comment]
@marfox Please correct me if I am wrong. I think that extracted wikipedia dump has a number of files. In each such file there are multiple documents each separated by </doc> tag. I thought that it would be better to extract only those documents from the file as output which have "doc ids" in soccer_ids.

**Comment by marfox** [comment]
On 3/13/15 2:59 PM, Abhishek Kumar Tiwari wrote:
> @marfox <https://github.com/marfox> Please correct me if I am wrong. I
> think that extracted wikipedia dump has a number of files. In each such
> file there are multiple documents each separated by tag.
This is the output of the third party lib we use.
Then, we further split by document.
> I thought that
> it would be better to extract only those documents from the file as
> output which have "doc ids" in soccer_ids.
That's what happens

**Pull request closed without merging**